src/cobald/monitor/format_line.py

```diff
@@ -68,6 +68,8 @@ def format(self, record: LogRecord) -> str:
             args = {}
         assert isinstance(args, Mapping), \
             'monitor record argument must be a mapping, not %r' % type(args)
+        assert all(elem is not None for elem in args.values()), \
+            'values for arguments are not allowed to be None'
```

Comment:
```suggestion
            'line protocol values must not be None'
```
I recommend rewording to make it clear it's not us disallowing it, but that line protocol cannot handle it.